.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-name: CI
+name: Linux AMD64 CI
```

Comment:
ci.yml is not for amd64

Reply:
AMD64 is a common synonym of x86_64.
Would you prefer to use `x86_64` and `aarch64` instead of `AMD64` and `ARM64` ?

Reply:
I thought it was "arm64".